src/views/RessourcesView.vue

```diff
@@ -1,29 +1,31 @@
 <script setup lang="ts">
+import codeLanguages from '@clubcapra/data/GithubStats.json';
```

Comment:
When importing a file, the import name should match the file name.
Rename the file `codeLanguages.json` or change the variable name to `GithubStats`.